file.go

```diff
@@ -322,3 +333,30 @@ func (f *File) setRecordTypes() {
 	}
 	f.Control.setRecordType()
 }
+
+// SetValidation stores ValidateOpts
+func (f *File) SetValidation(opts *ValidateOpts) {
+	if f == nil || opts == nil {
+		return
+	}
+
+	f.validateOpts = opts
+}
+
+// ValidateOpts returns Validation option
+func (f *File) ValidateOpts() *ValidateOpts {
+	return f.validateOpts
+}
+
+// IsArchiveTypeIndicator indicate to enable ArchiveTypeIndicator validation
+func (v *ValidateOpts) IsArchiveTypeIndicator() bool {
```

Comment:
I'm not sure what this helper is for. We should override the check, not offer getters.